### DISCUSSION TITLE
VS Code Syntax Support?

### DISCUSSION THREAD

**Discussion by c-fitzmaurice** [pestphp/pest#479] [General]
I'm just starting to expose Pest here and can't figure out a way to get proper syntax support?

VS Code keeps telling me `it` and `test` are not proper methods and flagging them...what am I missing?

Thanks!

**Comment by nelson6e65** [comment]
Are you using [PHPStan](https://phpstan.org)? That tool is already aware of these global signatures.

Maybe your vs-code plugin is not loading the global function's signatures.

Try to run `composer install` again and see if your plugin is loading the type-hints now.
